hack/init-buildx.sh

```diff
@@ -20,7 +21,7 @@ create_or_use_buildx_builder() {
 
   check_buildx
 
-  current_builder="$(docker buildx inspect "${builder_name}")"
+  current_builder="$(docker buildx inspect "${builder_name}" 2>/dev/null)" || echo "Builder '${builder_name}' not found"
```

Comment:
Done.

Reply:
Yes, you're right; it will have an empty value. However, we handle errors properly later in the script. I don't see any issues with either the failure or success case, so everything looks good.

Reply:
please correct me if i am wrong,
this is an expression that must success, hence we should fail fast (i.e remove the `|| echo ...`) ?

Reply:
No, by default, `docker buildx inspect <builder-name>` will print an error to the console if the builder is not found. Since we want to suppress this error, it's better to use `||` with an echo statement for debugging purposes.